api/src/main/java/org/apache/iceberg/FileStatistics.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg;
+
+import java.nio.ByteBuffer;
+import java.util.Map;
+
+public class FileStatistics {
+
+  private long recordCount;
+
+  private Map<Integer, Long> valueCounts;
+
+  private Map<Integer, Long> nullValueCounts;
+
+  private Map<Integer, Long> nanValueCounts;
+
+  private Map<Integer, ByteBuffer> lowerBounds;
+
+  private Map<Integer, ByteBuffer> upperBounds;
+
+  public long getRecordCount() {
```

Comment:
Iceberg follows its own naming conventions which is agreed by the community
https://iceberg.apache.org/contribute/?h=contribute#method-naming

Can LLM learn this? :D